app/views/laundry_rooms/index.html.erb

```diff
@@ -8,6 +8,7 @@
               <% if booking[:time] > (DateTime.now - 5.hours) %>
                 <ul>
                   <li><%= booking[:time].beginning_of_hour.to_formatted_s(:short) %></li>
+                  <%= link_to 'Unbook', laundry_room_delete_booking_path(laundry_room_id: @laundry_rooms.first, time: booking[:time]) %>
```

Comment:
Don't we have the LaundrRoom instance saved in a variable at this point? What is tha danger of hardcoding the instance to `@laundry_rooms.first`?

Reply:
Where do we have the LaundryRoom instance saved? 

What I can tell we only have all the laundry rooms set, in the LaundryRoom controller index action. Supporting multiple laundry rooms might be a different User story in the future, wouldn't you say? We did agree to this early in the design sprint. 

Please correct me if I'm misunderstanding your comment.

Reply:
Using booking.bookable_id